### ISSUE TITLE
DB init file not loaded

### ISSUE THREAD

**Issue by robert-annies** [daboe01/Clinical#1]
Hello,

I'm working with postgresql 9.5 (also tried with 9.2).

When loading the database init file with `cat sql_template.sql | pgsql aug_clinical` following error is shown by pgsql:

```
ERROR:  syntax error at or near "and"
LINE 1: and ( abs(day - ($2::text||'-'|| $1::text ||'-1')::date) < 6...
        ^
ERROR:  unterminated dollar-quoted string at or near "$_$;


ALTER FUNCTION public.calendar_function(integer, integer) OWNER TO postgres;
...
```

cheers
Robert

**Comment by daboe01** [comment]
hi @robokopp, thank you for spotting this. should be fixed with 309bc456d36efe8599db9a124adf195e679ae74d.

**Comment by robert-annies** [comment]
Thanks!
